README.md

```diff
@@ -10,6 +10,8 @@
 
 Documentation, Code Templates and a Showcase application to demonstrate features of the Natural Resources Common Document Generation Service
 
+To learn more about **Common Services** available including [hosted email](https://github.com/bcgov/common-hosted-email-service), [messaging](https://github.com/bcgov/nr-messaging-service-showcase) and [document generation](https://github.com/bcgov/document-generation-showcase) and to see them in action, visit the [Common Services Showcase](https://bcgov.github.io/common-service-showcase/) page.
+
```

Comment:
I would suggest advertising just "Common Services" without going into depth about the suite of available APIs currently as this list will grow over time. It would be better to just point them to the correct url and do the actual advertisement there instead -> we don't want to be needing to update this readme everytime we introduce a new Common Service.